### PR TITLE
assign shortnames to behaviors as supported by plone.behavior

### PR DESCRIPTION
**Pull request by jensens**
this was just missing.

### CODE REVIEW

File: plone/app/dexterity/behaviors/configure.zcml [hunk 1]
```diff
@@ -68,6 +75,7 @@
 
   <!-- Navigation root -->
   <plone:behavior
+      name="plone.namefromfilename"
```
Comment (thet) [inline plone/app/dexterity/behaviors/configure.zcml]: naming error

Comment (jensens) [inline plone/app/dexterity/behaviors/configure.zcml]: ?

**Comment by thet** [comment]
There is another branch adressing this, created by @jensens (you probably forgot about that). I tend to like the other one actually a bit more:
https://github.com/plone/plone.app.dexterity/commit/208bda212ead71e52420a868e15607b6d3b880c2

combined names are written with underscore in between, which is better for readability (not so for wrinting-speed). Also minimum plone.behavior version stated, some titles updated.

Otherwise +1 for merging this non-breaking change.

**Comment by jensens** [comment]
actually the shortnames are to simplify remembering the name also and who knows w/o consulting docs if its rich_text or richtext? at least for non native speakers this is hard. i tend to keep it as simple as possible and skip the _ 
also i thought I deleted the old branch.

**Comment by jensens** [comment]
ah, forgot to mention: for readability and meaning there is the title. the title changes in the other branch are worth a look.

**Pull request merged**